app/resources/payment_request_resource.rb

```diff
@@ -0,0 +1,22 @@
+class PaymentRequestResource
+  include Alba::Resource
+
+  attributes :submitter_id, :request_type,
+             :profit_cost, :allowed_profit_cost,
+             :travel_cost, :allowed_travel_cost,
+             :waiting_cost, :allowed_waiting_cost,
+             :disbursement_cost, :allowed_disbursement_cost,
```

Comment:
these costs are only relevant for NsmClaims so can use the conditional if clause for these fields too like you did for AssignedCounselClaim